docs/source/conf.py

```diff
@@ -166,8 +177,26 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-# TEMPORARILY DISABLED
-#html_static_path = ['_static']
+html_theme = 'pydata_sphinx_theme'
+html_theme_options = {
+    'external_links': [
+        {'url': 'http://www.aiida.net/', 'name': 'AiiDA Home'}
+    ],
+    'github_url': 'https://github.com/aiidateam/aiida-core',
```

Comment:
```suggestion
    'github_url': 'https://github.com/aiidateam/aiida-core',
    'twitter_url': 'https://twitter.com/aiidateam',
```

Reply:
this is just an optional cherry on the top, we can always add it later, if you can't be bothered to re-run all the tests again